chalicelib/checks/wrangler_checks.py

```diff
@@ -710,3 +712,54 @@ def patch_wfr_and_log(wfr, full_output):
             check.summary = 'No WFR patches run'
 
     return check
+
+
+@check_function()
+def validate_entrez_geneids(connection, **kwargs):
+    ''' query ncbi to see if geneids are valid
+    '''
+    check = init_check_res(connection, 'validate_entrez_geneids')
+    problems = {}
+    timeouts = 0
+    search_query = 'search/?type=Gene&limit=all'
```

Comment:
could add `&field=geneid` just to grab less info (it doesn't seem like any other fields are being used). we don't have a lot of targets and probably won't have a lot of genes for the near future, but still probably makes the request quicker. just a suggestion.